packages/kovan-faucets/src/ts/zrx_request_queue.ts

```diff
@@ -0,0 +1,43 @@
+import {ZeroEx} from '0x.js';
+import BigNumber from 'bignumber.js';
+import promisify = require('es6-promisify');
+import * as _ from 'lodash';
+import * as Web3 from 'web3';
+
+import {configs} from './configs';
+import {errorReporter} from './error_reporter';
+import {RequestQueue} from './request_queue';
+import {utils} from './utils';
+// HACK: web3 leaks XMLHttpRequest into the global scope and causes requests to hang
+// because they are using the wrong XHR package.
+// Issue: https://github.com/trufflesuite/truffle-contract/issues/14
+delete (global as any).XMLHttpRequest;
+
+const DISPENSE_AMOUNT_ZRX = new BigNumber(0.1);
+const QUEUE_INTERVAL_MS = 5000;
+const KOVAN_NETWORK_ID = 42;
```

Comment:
I expect this to be useful elsewhere in the codebase eventually. Mind moving into the `configs` file?